### DISCUSSION TITLE
Underwater with BIRP not OK on iOS, Would URP work?

### DISCUSSION THREAD

**Discussion by jelmer3000** [crest-ocean/crest#1117] [Q&A]
Hi 
I'm testing the free version of Crest using the built in render pipeline. I noticed that the underwater effect is not working properly on iPhone. Part of the effect seems to be flipped vertically.
I did find [this commit,](https://github.com/wave-harmonic/crest/commit/579b839a9ce833f2d17bd547eae6453dda38704a) but using that (old?) shader code gave some other errors.

So... Is there a fix for the underwater effect on BIRP? 
Or how would URP version behave? Is there a way I could try that, without paying 70 dollars right away?

**Comment by magenta404** [comment]
Not sure if URP will behave better than BIRP. I removed that code because Metal on macOS was fine. Is this the same issue as #192?
